app/scripts/components/dashboard/charts-container.jsx

```diff
@@ -12,7 +12,8 @@ const ChartsContainer = React.createClass({
     return (
       <div className="charts-container">
         <div className="charts-container-header">
-          <T k="charts.toggle.activate" />
+          <span className="tab-label">
+          <T k="charts.toggle.activate" /></span>
```

Comment:
Watch the indentation here -- `<T>` should be indented inside `<span>`, and the closing `</span>` should be on a new line.